app/serializers/user_notification_total_serializer.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+class UserNotificationTotalSerializer < ApplicationSerializer
+  attributes :username,
+             :unread_notifications,
+             :unread_personal_messages,
+             :unseen_reviewables,
+             :topic_tracking,
+             :group_inboxes
+
+  def initialize(object, opts)
+    super(object, opts)
+
+    @new_personal_messages_notifications_count = object.new_personal_messages_notifications_count
+  end
```

Comment:
It is a little odd to have to override the `initialize` method. Usually, we memoize by defining another method and then memoize the results when the method is called. Something like

```
def new_personal_messages_notifications_count
  @new_personal_messages_notifications_count ||= object.new_personal_messages_notifications_count
end
```